public/apps/configuration/panels/role-edit/role-edit.tsx

```diff
@@ -106,12 +106,12 @@ export function RoleEdit(props: RoleEditDeps) {
     }
   }, [addToast, props.action, props.coreStart.http, props.sourceRoleName]);
 
-  const [actionGroups, setActionGroups] = useState<string[]>([]);
+  const [actionGroups, setActionGroups] = useState<Array<[string, ActionGroupItem]>>([]);
```

Comment:
Just for my understanding, why are we needing this `ActionGroupItem`, or can we use the original `Action` here?

Reply:
If i'm understanding your question correctly I need the whole item here in order to tell whether it is a cluster/index one later on in order to filter them accordingly